lib/graphql/schema/directive/flagged.rb

```diff
@@ -7,7 +7,7 @@ class Directive < GraphQL::Schema::Member
       # In this case, the server hides types and fields _entirely_, unless the current context has certain `:flags` present.
       class Flagged < GraphQL::Schema::Directive
         def initialize(target, **options)
-          if target.is_a?(Module) && !target.ancestors.include?(VisibleByFlag)
```

Comment:
This spec broke when an interface was flagged because of how the implementations of `visible?` stacked in the inheritance chain. Making object types get this included on their own fixed it.